### DISCUSSION TITLE
API Reference?

### DISCUSSION THREAD

**Discussion by GeoffNordling** [mozman/ezdxf#1048] [Q&A]
Hi, so sorry if this is obvious and I missed it, but is there a complete API reference somewhere?

For example, if I want to know if ezdxf.readfile() has any optional parameters besides the file name? Or if I want to know the specific return type of some other function?

From searching, I did find the "internal data structures" here:
https://ezdxf.readthedocs.io/en/stable/develop/index.html#internal-data-structures

But that seems incomplete. It doesn't document ezdxf.readfile(), for example.

**Comment by mozman** [comment]
There is a reference section in the documentation and a search function.

**Reply by GeoffNordling** [reply]
I have found that I can pretty much type what I want into search function in top left. Thank you :D
